### PR TITLE
feat(styles): add delta theming for Dynamic Page [ci visual]

### PR DESCRIPTION
**Pull request by InnaAtanasova**
## Related Issue
Part of https://github.com/SAP/fundamental-styles/issues/3213

## Description
add delta theming for Dynamic Page

### CODE REVIEW

File: src/styles/theming/common/switch/_sap_fiori.scss [hunk 1]
```diff
@@ -70,4 +70,13 @@
   --fdSwitch_Compact_Semantic_Handle_Offset: -1.9375rem;
   --fdSwitch_Compact_Semantic_Active_Handle_Offset: 0.0625rem;
   --fdSwitch_Compact_Semantic_Active_Handle_Offset_Rtl: var(--fdSwitch_Active_Handle_Offset);
+
+  // Dynamic Page Layout
```
Comment (platon-rov) [inline src/styles/theming/common/switch/_sap_fiori.scss:74]: It's probably shouldn't be here, in switch styles.

Comment (InnaAtanasova) [reply src/styles/theming/common/switch/_sap_fiori.scss:74]: oh, nice catch! I didn't see the `switch` folder, just the `common` folder.  :D

**Review by platon-rov — COMMENTED** [review]
* font sizes are different, but should be the same, 0.875rem

Screenshot 2022-04-14 at 14 51 13

* I didn't find any example like this in wiki, but i assume that we're missing gap between items

image

**Review by mikerodonnell89 — CHANGES_REQUESTED** [review]
In quartz light, these buttons have lost their hover and down states

Screen Shot 2022-04-14 at 1 29 17 PM

**Comment by InnaAtanasova** [comment]
> * font sizes are different, but should be the same, 0.875rem
> 
> Screenshot 2022-04-14 at 14 51 13
> 
> * I didn't find any example like this in wiki, but i assume that we're missing gap between items
> 
> image

I don't think this is the correct PR :)

**Comment by InnaAtanasova** [comment]
> https://deploy-preview-3297--fundamental-styles.netlify.app

fixed

**Review by mikerodonnell89 — APPROVED** [review]

**Pull request merged**